content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Rapid Diagnostic Toolkit Integration"
+linkTitle: "RDToolkit Integration"
+weight:
+description: >
+  Integration for the Rapid Diagnostic Toolkit (RDToolkit)
+keywords: RDToolkit RDT
+
+---
+
+_Introduced in v3.12.0_
+
+CHT has an integration with [Rapid Diagnostic Toolkit (RDToolkit)](https://github.com/dimagi/rd-toolkit) which is an Android app developed by Dimagi that facilitates the use of Rapid Diagnostic Test (RDT) in global health care. 
+The integration is only available in the CHT Android app and consists of two Enketo widgets (rdtoolkit-provision and rdtoolkit-capture) that are used in XForms to provision RDT and capture results.
+
+## Create the XForms
+It is required to create one form for provisioning RDT and one form for capturing RDT results, see [the documentation](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/) for more details about app forms.
+
+### Provisioning RDT form
+The following labels are required in the “appearance” tag:
+* `rdtoolkit-provision`: To attach the Enketo widget that connects with the RDToolkit app.
+* `rdtoolkit-action-btn`: To attach the action button that provisions RDT, this button will open the RDToolkit app.
```

Comment:
I can't think of other examples that require two appearances, so I am curious why two appearance fields are needed here? What happens if only one is set?

Reply:
The `rdtoolkit-provision` attaches the widget and sets the scope of the widget, so we know where to display the preview and where to get the rdtoolkit exclusive fields (example: rdtoolkit_session_id field). 

The `rdtoolkit-action-btn` defines where the button to open RDToolkit app is going to display, for example we might want to display the button before a disclaimer label. If this is not provided, then the user won't be able to provide a RDTest.

Reply:
If omitting the `rdtoolkit-action-btn` would make it not work, it would be better to make the default button included with `rdtoolkit-provision`. If now or in the future we want to separate that out, we could add a second appearance value which makes it explicit, like `rdtoolkit-action no-button`.

Reply:
Done! I've removed the class and updated documentation and code.